src/state/queries/notifications/unread.tsx

```diff
@@ -136,7 +136,7 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
 
             // only fetch subjects when the page is going to be used
             // in the notifications query, otherwise skip it
-            fetchAdditionalData: !!invalidate,
+            fetchAdditionalData: true,
```

Comment:
If we fetch the additional data every time, we can filter out root quotes and all levels of reposts